staff/javier-lopez/beers-app/scripts/main.js

```diff
@@ -67,23 +63,15 @@ ResultsList.prototype.onItemClick = function (callback) {
  * 
  * @param {string} title The item title
  * @param {string} info The information about an item
- * @param {[number]} coords The geodesic coordinates for google maps
+ * @param {[number]} abv The graduation of the beer
```

Comment:
If you put as parameters (title, abv, description), try to put in the comments in the same order. Then, the person who reads your code could understand better what you are using below. : D